### ISSUE TITLE
Set up a Competitive Intelligence WG

### ISSUE THREAD

**Issue by pmoorman** [rchain/bounties#776]
### SMART Goal

Creation of a Competitive Intelligence working group (or similar structure) to organize CI gathering and processing for RChain.

###  Status

- The WG is formed, and composed of the following 5 people: Dror D., Jeremy B., Jonathan K., Patrick M., and myself.

- We've had our first meeting, this Monday

- _More status updates will follow here... it's a bit early to call this a success just yet_

### Turning CI into bounty tasks

I'd like to still work on this issue and see if we can turn the CI process into something that the bounty program to help with. Our first focus will be on getting the WG going (in-process right now), and then we'll see how we can open this up for more people to contribute effectively.

_Work executed for the Competitive Intelligence WG should probably be tracked in a different issue._

**Comment by dckc** [comment]
I sure wish you would create specific issues and generalize later. What does "getting the WG going" mean? Meeting for the sake of meeting?

**Comment by pmoorman** [comment]
Hopefully not!

The first effort was getting a group of people together that want to help with this, and working out how to organize things between the coop, the marketing org, and the bounty system. Credit to @patrick727 and @jonathan (what's his Github handle?!) for helping with that.

We now have a place to put all the stakeholders together, we have an inventory of what's actually needed, and we have a process in place. I haven't checked with the others if I can share the meeting notes, so I'll wait till the next call to discuss that. 

For now, the most immediate strategy is to do sprints focused on one competitor at a time, with Cardano being the first one. 

The next step is —once we have this going between the 5 of us— to make this more open to contributions from all RAMs

**Comment by dckc** [comment]
>  we have an inventory of what's actually needed

Where? What did I miss?

> What do you mean with "create specific issues and generalise later?

A specific issue would have, for example,  Cordano in the title.

p.s. Jonathan is @ottermagically IIRC

**Comment by pmoorman** [comment]
**Where** => in the meeting minutes, but I'll have to check with everyone whether we can publish them. Will check that next call (which is on Tuesday), and then update here.

**As for specificity:** I'll try my best to spin work like this into issues that all RAMs can work on, but it's best if we can just get started with 5 people to define the process, then invite everyone to collaborate once we have a bit more structure. 

In other words:  I'm working on it!

**Comment by dckc** [comment]
I look forward to those meeting minutes -- or at least the inventory of what's actually needed.

Meanwhile, while working on arch doc review / revision today, I ran across the [comparison of blockchains][1] which looks like it could use some love. Doesn't mention REV as a protocol token, for example.

[1]: http://arch-doc.rchain.coop/en/latest/introduction/comparison-of-blockchains.html

**Comment by dckc** [comment]
I studied [diro](https://www.diro.io/) tonight, per suggestion from @ottermagically.

In sum: very strong identity story.

I wonder about privacy, though. Let's see if they answer https://twitter.com/dckc/status/1011489718901641216

**Comment by pmoorman** [comment]
### Update

Here's a short update regarding the activities of the Competitive Intelligence WG.

- There have been concerns about what information from the competitive intelligence WG needs to remain confidential and not be open to "the general public". Until we have everyone in the WG agreed on the approach on this, we can't publish minutes / meeting notes (yet).

- The aim of the WG is to provide helpful competitive intelligence information to the entire RChain community. That includes the marketing org (so they can write blog posts comparing "platform X vs. RChain", etc.), business development people such as @ottermagically, partners like Reflective, but also just everyone in the community, as they help out with marketing and other tasks.

- Of course, you can't do that while holding all your cards to your chest, so we're working on opening up. As per the above, this is a bit more delicate than I foresaw, and will take a bit longer.

- The initial platform we're digging into is Cardano, because they are one of the biggest players closest to what RChain does. Further platforms will be determined based on what we get asked about the most.

- @ddayan, @jeremybeal11 and myself are currently also working out the infrastructure to do comparisons on many more platforms and compare them side-by-side in a big spreadsheet (or probably Airtable, actually). This will provide foundation for other people to assist with competitive intelligence research.

- Finally, I hope that we'll have better updates in a couple weeks from now, once we figure out the balance between confidentiality & inclusiveness.

**Comment by pmoorman** [comment]
## Update (part II)

We have now confirmed with everyone from the WG that it's fine to actually share publicly what's going on. The only area we'll keep restricted are the work-in-progress documents that need to be turned into marketing collateral like blog posts, etc.

Find here the meeting notes from the last 3 calls we've had (note: I didn't write them expecting them to be publicly scrutinized, but will publish here in a bid for transparency):

- [Meeting I, June 12th](https://docs.google.com/document/d/1vQ5AGhG_QaiS3w7EFjROR8a6SxgVz_go6rJ7Rr_Zkpo/edit?usp=sharing)
- [Meeting II, June 19th](https://docs.google.com/document/d/1gvdYVnMfRvJEJckqUcXYMfv3oHjmxua5X8Lf7JiTJRk/edit?usp=sharing)
- [Meeting III, June 25th](https://docs.google.com/document/d/16lQZaZnyKRbflkwe09XhA2WJYqkH5582DdNG_35YhPo/edit?usp=sharing)

Right now, all documentation lives in a Google Drive folder, but we're probably moving things over to combine Google Drive and Airtable (a sort of fancy spreadsheet tool). It still has access restrictions on it, but we'll talk on next calls on how to open that up.

For now, the WG is focused on researching Cardano, and using Cardano as a 'test case' to specify what we actually need in terms of researching a competitor. We'll move on to next platforms in ~2 weeks probably, and by that time I hope we can open up to more RAMs to help out.

**In short, my idea is:**

- We start working in the open
- @ddayan, @jeremybeal11 and myself work out a template of sorts while researching Cardano
- Once Cardano is done, we open up to the bounty community, and say _"Hey, we need the same stuff as for Cardano, but for <platform X>"_
- For now the marketing org (esp. Jeremy) will take care of spinning blog posts and other material out of the research, but maybe in the future other people can help with this, too.

**Comment by tucsonblockchain** [comment]
https://docs.google.com/document/d/1U8wAPG6_Myk4oRPN8CtZRD0WQWi99Y3HRb7fJZvkUu0/edit?usp=sharing Created this doc, certainly unfinished, but @allanc mentioned this issue. Would like any thoughts, or if anybody would like to contribute and arrive at next steps, if worthy. I can take criticism good or bad but carved out what was asked and did best I could. Just don't know the audience and thought others were going to join in. Still can! Thx

**Comment by pmoorman** [comment]
Hey @tucsonblockchain I think everyone would be glad to have any apps on the network that can stress the system. That's really a great problem to have.

This issue was created with a goal of executing comparisons of different blockchains, and in that sense I don't really understand how your proposal fits under it. I would say that either it deserves it's own issue (probably), or maybe we could somehow create feature-parity apps on RChain and other networks (CryptoKitties vs. RhoKitties, or something of the like), and sort of have a "real life comparison". That second approach would be in line with this issue, but I'm not sure it's what you have in mind.

As an aside, you might want to contact Philipp from Inblockio (@phil on Discord I think). I have been told he's "broken" Ethereum before, and is eager to stress test RChain, too!

So to conclude:

1. Your line of thinking deserves it's own issue, rather than live here in the shadow of something else

2. In line with this issue, I can imagine it would be interesting to create Rho-kitties or something like that to create a Apples-to-Apples comparison of different networks. -- that's where I can see this issue and your ideas connect.

3. Connect with @phil on Discord, as he is one of the most qualified people to take these ideas further.

Hope that helps!

**Comment by AyAyRon-P** [comment]
Hi @pmoorman 

Is this still ongoing? Are there any updates? Do we need to vote on this for the month of August? 

If so please place the ONGOING: tag to the front of the project. Let me know what your thoughts are on budget for the month so we can vote accordingly.

**Comment by dckc** [comment]
I still do competitive intelligence from time to time and I would appreciate an opportunity to discuss it in something like a working group. So I hope this gets refined. (Or somebody let us know where the work is happening, if it's not reported / coordinated here.)

Meanwhile...

I'm not a fan of the ONGOING pattern. It's OK in exceptional situations, but those should be clearly justified. See for example, #208 where I added a bunch of stuff under "How will we measure completion?":
> Rather than measure completion, we aim to measure continued benefit worth keeping this process going.  ...

**Comment by pmoorman** [comment]
Hey all, 

it's about time I'd write an update here, regarding the status of this issue.

Earlier this summer we formed a tentative working group to work on competitive intelligence and comparisons to other platforms. In the progress, we found that there was primarily a need to have a sort of performance "baseline" of RChain itself, before it can helpfully be compared to other platforms. 

Furthermore, marketing has become careful (and rightfully so) about explicitly comparing RChain against other platforms in the space in the content they produce.

At the moment, there is no (bounty) work on competitive intelligence going on, but there's been extensive work by me and others to at least bridge the gap between value propositions on the one hand, and the "bare metal" of the technology on the other hand. This is an ongoing topic, and something we'll track primarily under website copywriting an such related topics.

I would consider the scope of this issue completed, and close the issue accordingly. @dckc (or others) if you want to discuss work related to this topic, feel free to reopen again, or reach out to either @ddayan, Jemery Beal (marketing org), Jonathan or me personally. I think that's the best we can do for now.  I'm expecting we might re-open this issue in a few months when the testnet is providing a better base for comparison against other platforms.
